python/covid_web/covid_web/rest.py

```diff
@@ -79,7 +82,34 @@ def get_app(self):
         return self
 
     def get_available_predictions(self):
-        return jsonify(self.available_predictions)
+        result = []
+        for prediction in self.available_predictions:
+            prediction['link'] = url_for("covid19_get_specific_prediction", country=prediction['country'], prediction=prediction['prediction'])
```

Comment:
The CI will tell you to format this line :)

Reply:
Changed.